packages/cli/lib/utils.js

```diff
@@ -140,6 +140,84 @@ exports.validateNotExisting = function(projDir) {
   return true;
 };
 
+/**
+ * validate source key or foreign key for relations
+ */
```

Comment:
Call `/* istanbul ignore next */` here instead?

Reply:
Right, it's so much better, thanks!